webui/package.json

```diff
@@ -21,7 +21,8 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "^4.0.1",
     "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "immer": ">=8.0.1"
```

Comment:
don't we need react-scripts that depends on this to do this?

Reply:
followed the recommendation, but I also prefer to upgrade react and not dependencies - closing this one. thanks.